### DISCUSSION TITLE
Using slot 1 and slot 2 simultaneously

### DISCUSSION THREAD

**Discussion by tim2000s** [NightscoutFoundation/xDrip#2923] [General]
As per https://github.com/NightscoutFoundation/xDrip/discussions/2834#discussioncomment-6343219

I'm attempting to run slot 1 and slot 2 on different phones. 

Slot 2 connected to Dexcom BYODA works with no  issues. 

Slot 1 to xDrip on the other phone won't establish a Bluetooth connection. (xDrip is attempting to connect, but doesn't seem to be able to pair. 

The log shows a "No valid timestamp for transmitter" error in DexTimeKeeper.

(Opened as discussion rather than Issue, as requested).

**Comment by Navid200** [comment]
If you turn off the other phone, does xDrip connect?

**Comment by tim2000s** [comment]
It attempts to pair, but the pairing fails.
![20230703_145329](https://github.com/NightscoutFoundation/xDrip/assets/14833174/6e2d004a-8a14-4bdc-8486-df4449596925)

**Reply by Navid200** [reply]
Therefore, the problem is not with the slot setting in xDrip.
Can you disable engineering mode?  And turn off the other phone.  Does xDrip connect now?
If not, you need to go over all settings to make sure they are correct.
https://navid200.github.io/xDrip/docs/G6-Recommended-Settings.html

If the phone, with xDrip on it, is Samsung, have you seen this?
https://github.com/NightscoutFoundation/xDrip/issues/2917

As long as xDrip is not operating properly with engineering mode disabled, it is pointless to focus on the slots.
